packages/docs-site/src/content/docs/docs/integrations/astro.mdx

```diff
@@ -8,11 +8,11 @@ import AstroLogo from "~icons/logos/astro";
 import TabbedCode from '@/components/TabbedCode.astro';
 import BugReportLink from '@/components/BugReportLink.astro';
 
-At DMNO we _love_ Astro (this very site is built with it). That's why we're very excited to make it easier (and safer) to use environment variables in all of your Astro apps, whether it's on the client or the server!
+At DMNO we _love_ Astro. This very site is built with it! That's why we're very excited to make it easier and safer to use environment variables in all of your Astro apps, whether it's on the client or the server.
 
 ## Setup
 
-### Install the dmno astro integration
+### Install the `dmno` astro integration
```

Comment:
I do like that we are being consistent, but unclear to me in some situations like this which to go for?

Reply:
yeah I could go either way on `dmno` vs DMNO, typically `dmno` should be specific to anything involving the `dmno` package, if that makes sense?